flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/physical/stream/SimplifyWindowTableFunctionRules.java

```diff
@@ -1,348 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.flink.table.planner.plan.rules.physical.stream;
-
-import org.apache.flink.table.planner.plan.nodes.physical.stream.StreamPhysicalCalc;
-import org.apache.flink.table.planner.plan.nodes.physical.stream.StreamPhysicalExchange;
-import org.apache.flink.table.planner.plan.nodes.physical.stream.StreamPhysicalWindowDeduplicate;
-import org.apache.flink.table.planner.plan.nodes.physical.stream.StreamPhysicalWindowJoin;
-import org.apache.flink.table.planner.plan.nodes.physical.stream.StreamPhysicalWindowRank;
-import org.apache.flink.table.planner.plan.nodes.physical.stream.StreamPhysicalWindowTableFunction;
-
-import org.apache.calcite.plan.RelOptRule;
-import org.apache.calcite.plan.RelOptRuleCall;
-import org.apache.calcite.plan.RelOptRuleOperand;
-import org.apache.calcite.rel.RelNode;
-
-import java.util.Collections;
-import java.util.List;
-
-/**
- * Planner rule that tries to simplify emit behavior of {@link StreamPhysicalWindowTableFunction} to
- * emit per record instead of emit after watermark passed window end if time attribute is event-time
- * and followed by {@link StreamPhysicalWindowRank} or {@link StreamPhysicalWindowJoin}.
- */
-public interface SimplifyWindowTableFunctionRules {
```

Comment:
why no exec plans are change after this rule is deleted?

Reply:
Those rules are used to attach a flag on window TVF which is followed after Window TopN/ Window Deduplicate/ Window Join. For those window TVF with flags would be normally executed in ExecNode, for other window TVF without flags would be thrown out in ExecNode.

Since we supported individual window table-valued function in planner, there is no need to distinguish window tvf based on the flags. So we removed those rules.

And there are some test which failed before could work now. For example, `WindowTableFunctionTest#testUnsupported`.